pages/tutor/dashboard.py

```diff
@@ -0,0 +1,47 @@
+"""The dashboard (course picker) page object."""
+
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from pages.tutor.base import TutorBase
+
+# from pages.tutor.course import TutorCourse
+
+
+class TutorDashboard(TutorBase):
+    """Tutor dashboard page object."""
+
+    URL_TEMPLATE = '/dashboard'
+    _root_locator = (By.CLASS_NAME, 'tutor-root')
```

Comment:
You will want to separate the class constants ('constant' is relative in Python) from the locators (e.g. a blank line between the two sections).